examples/shaders/shared/src/lib.rs

```diff
@@ -40,7 +40,7 @@ pub struct ShaderConstants {
 }
 
 pub fn saturate(x: f32) -> f32 {
-    x.max(0.0).min(1.0)
+    x.clamp(0.0, 1.0)
 }
```

Comment:
Wait, didn't this use to be weird? Or am I thinking of an interpolation method?
(I really like `clamp`, would've been better if it took `RangeInclusive` because the `..=` syntax is unambiguous, but this looks fine too)

Reply:
There was something about `x.step(y)` being intuitively reversed, are you thinking of that?

Reply:
Nope, pretty sure it was `lerp`.